recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
Heh, turns out that was the whole reason i started this, to see if i could track down why certain things fail in circle/appveyor, as I am using the casper/phantom approach elsewhere.

Why would we not want to run these?